src/adapters/leasing-adapter.ts

```diff
@@ -132,6 +132,23 @@ const getContactByContactCode = async (
   }
 }
 
+const getTenantByContactCode = async (
+  contactCode: string
+): Promise<AdapterResult<any, unknown>> => {
+  try {
+    const res = await axios.get(
+      `${tenantsLeasesServiceUrl}/tenant/contactCode/${contactCode}`
```

Comment:
`tenants` to align with DIGG-standard

Reply:
https://github.com/Bostads-AB-Mimer/onecore-core/pull/125/commits/d4edf8ca7d68f41eddd72fc5c31daef74786150b